frontend/src/reducers/dashboard/index.spec.js

```diff
@@ -0,0 +1,16 @@
+import {
+  RECEIVE_FAILURE_COUNT
+} from 'actions'
+
+import reducer from 'reducers/dashboard'
+
+describe('reducers/dashboard', () => {
+  describe('for RECEIVE_FAILURE_COUNT', () => {
+    it('sets overviewVisible for the specific event to true', () => {
+      const currentState = {}
+      const action = { type: RECEIVE_FAILURE_COUNT, data: { count: 8 } }
```

Comment:
just noticed that you parsed the count in the reducer... >_<
choose one an go for it. I don't have a strong opinion. Just need to update the action spec then.